dm/ctl/master/start_task.go

```diff
@@ -48,6 +49,20 @@ func startTaskFunc(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	// If task's target db is configured with tls certificate related content
+	// the contents of the certificate need to be read and transferred to the dm-master
+	task := config.NewTaskConfig()
+	yamlErr := task.RawDecode(string(content))
+	if yamlErr != nil {
+		return yamlErr
+	}
+	if task.TargetDB.Security != nil {
```

Comment:
panic because `task.TargetDB` is nil

as in https://github.com/pingcap/dm/pull/1910#discussion_r686457014